lib/state/app_state.dart

```diff
@@ -10,6 +11,8 @@ class AppState with _$AppState {
   factory AppState({
     @Default('') String sample,
     @Default(<Pokemon>[]) List<Pokemon> pokemons,
+    @Default(PokemonDetails(height: 0, weight: 0, baseExperience: 0, abilities: [], moves: [], stats: []))

```

Comment:
make this nullable instead with null as its default value 🙂

Reply:
Resolved po sir